tools/packaging/kata-deploy/local-build/dockerbuild/Dockerfile

```diff
@@ -21,6 +21,7 @@ RUN apt-get update && \
         docker.io \
         curl \
         make \
+        cmake \
```

Comment:
Hmm. You're adding the `cmake` here to build dragonball, right?

I wonder if the addition shouldn't be to https://github.com/kata-containers/kata-containers/blob/main/tools/packaging/static-build/shim-v2/Dockerfile instead.